sourcegraph/sourcegraph
# [SG-46115] - Merge the `search-ui` package into the `branded` package

## Description
Colocate the files from the `client/search-ui` package directories into `client/branded`.

## Refs
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/46115)
[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SG-46115)

## Test Plan
1. Run web and vscode app
2. Verify that the `search-ui` components are preserved.
3. Verify in the codebase that the `search-ui` and `branded` packages has been merged.

## App preview:

- [Web](https://sg-web-contractors-sg-46115.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
